modules/albo-installation.adoc

```diff
@@ -0,0 +1,282 @@
+// Module included in the following assemblies:
+//
+:_mod-docs-content-type: PROCEDURE
+[id="aws-load-balancer-operator-installation_{context}"]
+= Installing the AWS Load Balancer Operator
+
+After setting up your environment with your cluster, you can install the AWS Load Balancer Operator using the CLI.
+
+.Procedure
+. Create an AWS IAM policy for the AWS Load Balancer Controller:
++
+[NOTE]
+====
+The policy is sourced from link:https://raw.githubusercontent.com/kubernetes-sigs/aws-load-balancer-controller/v2.4.4/docs/install/iam_policy.json[the upstream AWS Load Balancer Controller policy] plus permission to create tags on subnets. This is required by the Operator to function.
+====
++
+[source,terminal]
+----
+ $ oc new-project aws-load-balancer-operator
```

Comment:
Might be best to separate the act of making a project from the act of creating a policy?

Likewise, probably best to clarify what the steps are here versus following MOBB shorthand, just in case one of these ceases to work in future.

Reply:
Broke that up with 
![image](https://github.com/user-attachments/assets/e26628aa-04cf-4d26-945b-1cd51cf441ec)